.gitlab/ci/sdk-nightly.yml

```diff
@@ -93,13 +100,9 @@ demisto-sdk-nightly:create-instance:
 demisto-sdk-nightly:run-commands-against-instance:
   extends:
     - .default-job-settings
-    - .sdk-nightly-schedule-rule
+    - .sdk-nightly-schedule-rule-always
   variables:
-    DESTROY_INSTANCES: "true"
     INSTANCE_ROLE: "Server Master"
-  needs: ["demisto-sdk-nightly:create-instance"]
```

Comment:
why is this not needed anymore?

Reply:
By default all jobs of a previous stage need to finish before a job in a subsequent stage start.